requirements.txt

```diff
@@ -39,7 +39,6 @@ tqdm>=4.66.2
 transformers>=4.30.0
 trl>=0.7.11
 wandb>=0.16.4
-langchain-text-splitters
 # do not use 8.4.0 due to a bug in the library
 # https://github.com/instructlab/instructlab/issues/1389
 tenacity>=8.3.0,!=8.4.0
```

Comment:
This can go away if `langchain-text-splitters` goes away.

Reply:
It's being pulled by `langchain-text-splitters`.

Reply:
OK, that will need to wait for a version of `instructlab-sdg` with this requirement in place.  langchain moved there, but not this part because I didn't realize it was related.

Reply:
https://github.com/instructlab/sdg/pull/38

Reply:
done